src/wallet/wallet.cpp

```diff
@@ -37,7 +37,36 @@
 #include <boost/algorithm/string/replace.hpp>
 #include <boost/thread.hpp>
 
-std::vector<CWalletRef> vpwallets;
+static std::vector<std::shared_ptr<CWallet>> vpwallets;
```

Comment:
Should `typedef std::shared_ptr<CWallet> CWalletRef`

Reply:
You mean also `typedef std::shared_ptr<CWallet> CWalletRef`?